### PR TITLE
fix kubevirt-web-ui tag when docker_tag is configured

### PR DESCRIPTION
**Pull request by SchSeba**
**What this PR does / why we need it**:
This PR remove the "v" character from the `set_fact: kubevirt_web_ui_image_tag`

This PR also fix the deprovision of kubevirt. when we check the kubevirt namespace
oc get ns | grep -w 'kubevirt'
kubevirt                            Active    1d
kubevirt-web-ui                     Active    1h

check the command to be 
oc get ns | grep 'kubevirt '  (space added)

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
None
```

---
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/kubevirt-ansible/535)

### CODE REVIEW

**Comment by SchSeba** [comment]
@cynepco3hahue can you please take a look :)

File: roles/kubevirt/tasks/deprovision.yml [hunk 1]
```diff
@@ -65,8 +65,11 @@
 - name: Delete KubeVirt Resources
   command: "{{ cluster_command }} delete -f /tmp/kubevirt.yaml --ignore-not-found=true --wait=false"
 
+- name: Delete Kubevirt Namespace
+  shell: "{{ cluster_command }} delete ns {{ namespace }} --ignore-not-found"
+
 - name: Wait until {{ namespace }} namespace will dissappear
-  shell: "{{ cluster_command }} get ns | grep -w {{ namespace }}"
+  shell: "{{ cluster_command }} get ns | grep '{{ namespace }} '"
```
Comment (dankenigsberg) [inline roles/kubevirt/tasks/deprovision.yml]: how about doing this more precisely: 
```
  ! {{ cluster_command }} get ns {{ namespace }} 
```

Comment (SchSeba) [reply roles/kubevirt/tasks/deprovision.yml]: Good one I change it now

Comment (dankenigsberg) [reply roles/kubevirt/tasks/deprovision.yml]: oops, no need for the leading "!"

**Comment by SchSeba** [comment]
the Failed look unrelated.
https://jenkins.ovirt.org/job/kubevirt_kubevirt-ansible_standard-check-pr/2116//artifact/check-patch.openshift_3-11.el7.x86_64/mock_logs/script/stdout_stderr.log

**Comment by SchSeba** [comment]
ci test please

**Comment by SchSeba** [comment]
ci test please

**Comment by myakove** [comment]
LGTM

**Comment by cynepco3hahue** [comment]
ci test please

**Comment by cynepco3hahue** [comment]
hm Sanity test fails, possible that it starts test before all components are running.

**Comment by SchSeba** [comment]
I try to add a sleep before integration tests start. lets check if this fix the issue

**Comment by SchSeba** [comment]
for reference 
```
TASK [kubevirt : Download KubeVirt Template] ***********************************
Monday 07 January 2019  13:20:32 +0000 (0:00:00.219)       0:32:40.912 ******** 
fatal: [localhost]: FAILED! => {
    "changed": false, 
    "dest": "/home/jenkins/workspace/kubevirt_kubevirt-ansible_standard-check-pr/kubevirt-ansible/roles/kubevirt/templates/kubevirt.yaml.j2", 
    "state": "absent", 
    "url": "https://github.com/kubevirt/kubevirt/releases/download/v0.12.0-alpha.2/kubevirt.yaml.j2"
}

MSG:

Connection failure: ('The read operation timed out',)
```

This looks unrelated to this PR I will rerun the tests

**Comment by SchSeba** [comment]
ci test please

**Comment by SchSeba** [comment]
@gbenhaim can you help me here with the ci?
I don't think the fail is related to this PR

**Comment by SchSeba** [comment]
ci test please

**Comment by marchukov** [comment]
ci test please

**Comment by SchSeba** [comment]
The tests pass now Can we merge it @cynepco3hahue?

**Pull request merged**